addons/mail/models/mail_thread.py

```diff
@@ -1899,7 +1899,8 @@ def _message_post_process_attachments(self, attachments, attachment_ids, message
     def message_post(self, body='', subject=None,
                      message_type='notification', subtype=None,
                      parent_id=False, attachments=None,
-                     notif_layout=False, notif_values=None, **kwargs):
+                     notif_layout=False, add_sign=False,
+                     mail_auto_delete=True, model_description=False, **kwargs):
```

Comment:
Small note: just move model_description with add_sign to have all notif parameters on the same line